config/hardware/addons/lights/caselight.cfg

```diff
@@ -0,0 +1,26 @@
+#####################################################################
+#   LED Control
```

Comment:
Can modify to "neopixel LEDs caselight control" ?

Also can you change the name of the file to `neopixel_caselight.cfg` ?

Reply:
Ok done